src/test/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:h2://mem:rpg_test;DB_CLOSE_DELAY=-1
```

Comment:
is `;DB_CLOSE_DELAY=-1` commented?

Reply:
I started using a Postgres db to run tests.